server/controllers/token.json

```diff
@@ -0,0 +1 @@
+{"access_token":"ya29.a0ARrdaM-uLYdKFXRLs8ZQMimZLV5vICRzQZrM4oFbbR5RQHbwH5rq38ZqhhH0PAGSs2y8jswiUd8vKZRIHyr7spumyKTrimFINCV2UuaL0M1A3C51zS_zE_706QYwBinxn3vF-7F_FtqVMGqKMCZFavkKKt67","refresh_token":"1//0f96ZoC6_LsXgCgYIARAAGA8SNwF-L9IrO1vU1bVQK2nyH9uItYNtE2PctbfBOFj7SnVYBlXe7uvLmDUNftBY0OIdP3QeoxheiIY","scope":"https://www.googleapis.com/auth/calendar.readonly","token_type":"Bearer","expiry_date":1627704704963}
```

Comment:
please put in git ignore to avoid exposing public data

Reply:
done!